### ISSUE TITLE
Zigbee2Mqtt doesn't load overview of devices 

### ISSUE THREAD

**Issue by jilles** [stas-demydiuk/domoticz-zigbee2mqtt-plugin#468]
![image](https://user-images.githubusercontent.com/196457/99148971-467f7580-268b-11eb-9427-46db667d54be.png)

* I've rebooted > no luck
* disable/enable plugin > no luck

Please let me know where else to look to fix or analyse this.

Thanks!

PS love the plugin!

**Comment by Bartekn86** [comment]
journalctl -u zigbee2mqtt.service -f
mqtt is working ?

**Comment by jilles** [comment]
Yes everything works fine, meaning that all connected Zigbee devices work and domoticz get updated.

**Comment by jefdemas** [comment]
In My case it sometimes shows up after waiting for several minutes.

**Comment by jilles** [comment]
Nothing after atleast 9 minutes...

**Comment by jilles** [comment]
I've installed the front-end of zigbee2mqtt. This does work.

**Comment by jilles** [comment]
I've reïnstalled the plugin and added all devices one by one. Overview worked, until I added some Hue filament bulbs. Last time these were also the last to be added. Can this somehow be related or am I chasing ghosts? As said Zigbee2mqtt fronted still works.

**Comment by radekson** [comment]
I have the same problem.
Plugin: 0.2.1
Domoticz: 4.11415
Python: 3.5.3
zigbee2mqtt: 1.16.1

**Comment by jilles** [comment]
So I finally realised I should have a look at the console of my browser (ctrl+shift+i) which should me I had some websocket errors:
![image](https://user-images.githubusercontent.com/196457/100624164-623d7980-3323-11eb-9209-455a19c3191c.png)

I realised this must have something to do with the fact I use reverse proxy. So I then browse to this page without reverse proxy: 

![image](https://user-images.githubusercontent.com/196457/100624383-b34d6d80-3323-11eb-8046-916eb909e18f.png)

So wasn't plugin related at all. I added some headers to my nginx config (within my Synology DSM)
- CLI: https://github.com/orobardet/dsm-reverse-proxy-websocket
- In UI: 
![image](https://user-images.githubusercontent.com/196457/100624615-edb70a80-3323-11eb-8ba5-8cd95ea8149c.png)

And now everything works again. So I'm closing this issue.
@radekson I hope this helps you as well.
